src/Orchard.Specs/Hosting/Orchard.Web/Web.config

```diff
@@ -141,4 +141,22 @@
             <mimeMap fileExtension=".woff2" mimeType="font/woff2" />
         </staticContent>
     </system.webServer>
+    <runtime>
+        <gcServer enabled="true" />
+        <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+            <probing privatePath="App_Data/Dependencies" />
```

Comment:
The removal of this line during the MVC upgrade was the root cause behind the error I mentioned under the issue:

> None of the constructors found with 'Autofac.Core.Activators.Reflection.DefaultConstructorFinder' on type 'Orchard.Environment.DefaultOrchardShell' can be invoked with the available services and parameters:
> Cannot resolve parameter 'Orchard.IWorkContextAccessor workContextAccessor' of constructor 'Void .ctor(Orchard.IWorkContextAccessor, System.Collections.Generic.IEnumerable1[Orchard.Mvc.Routes.IRouteProvider], System.Collections.Generic.IEnumerable1[Orchard.WebApi.Routes.IHttpRouteProvider], Orchard.Mvc.Routes.IRoutePublisher, System.Collections.Generic.IEnumerable1[Orchard.Mvc.ModelBinders.IModelBinderProvider], Orchard.Mvc.ModelBinders.IModelBinderPublisher, Orchard.Tasks.ISweepGenerator, System.Collections.Generic.IEnumerable1[Orchard.Owin.IOwinMiddlewareProvider], Orchard.Environment.Configuration.ShellSettings)'.

Some of the assembly binding redirects also had to be restored for the test app to be able to start.